### ISSUE TITLE
Doesn't work with playwright 1.31 and lighthouse 9

### ISSUE THREAD

**Issue by grybykm** [abhinaba-ghosh/playwright-lighthouse#47]
I'm no longer able to run `lighthouse` because of: `Cannot create new tab, and no tabs already open.` since I've upgraded the playwright to v1.31. I'm still using lighthouse v9.

Maybe it is somehow related to chromium update or https://github.com/GoogleChrome/lighthouse/issues/14736.
I haven't tried switching to lighthouse v10 because after upgrading the lighthouse I'm getting:
```
    import log from 'lighthouse-logger';
    ^^^^^^

    SyntaxError: Cannot use import statement outside a module
```

Did anyone manage to upgrade to the latest playwright version and keep their lighthouse tests running?

**Comment by yardenhochman** [comment]
I have the same issue

**Comment by grybykm** [comment]
There are two ways to deal with this one:
- build a wrapper package around lighthouse and playwright-lighthouse to convert these packages from type module to normal ones. I have actually built one but decided to choose another 
- upgrade the project to use type module

First, I built and published to npm a wrapper as a workaround but decided not to proceed further with it.

Finally, I've chosen the second option even though I had to change ALL the files in my project as far as it is more feature oriented.
